src/integration/pages/articles/afrique/__snapshots__/amp.test.js.snap

```diff
@@ -3,3 +3,7 @@
 exports[`AMP Articles Gist list item should match text 1`] = `"Lorem ipsum dolor sit amet, consectetur adipiscing elit."`;
 
 exports[`AMP Articles Gist title should match text 1`] = `"Points clés"`;
+
+exports[`AMP Articles Image Copyright BBC should match snapshot 1`] = `null`;
```

Comment:
We shouldn't have null snapshots, as it indicates there's a problem with the test